LinuxWallpaperEngineSettings.qml

```diff
@@ -130,35 +130,63 @@ PluginSettings {
             layer.enabled: true
         }
 
-        CachingImage {
+        AnimatedImage {
```

Comment:
Aesthetically, only comment I have is this causing the image to be way too long (a square essentially) - which not only clips the image but just makes scrolling through the settings a bit annoying.

Reply:
You are right - I did notice the same thing on my wallpapers after checking. But the crop I feel like is justified at the expense of avoiding it taking up half the real estate. Either that, or maybe remove the preview entirely?